apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -713,7 +713,7 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                     label={ fieldName }
                     required={ schema.required }
                     requiredErrorMessage={ fieldName + " " + "is required" }
```

Comment:
Missing i18n string in `requiredErrorMessage` & `placeholder`. Let's add a TODO comment and address in a different PR.

Reply:
Since this issue is there in the whole component I will track and address with a different PR

Reply:
Great!